### DISCUSSION TITLE
ImageResponse.urlResponse is nil mostly

### DISCUSSION THREAD

**Discussion by seanchelulu** [kean/Nuke#680] [Q&A] [answered]
ImageResponse.urlResponse is nil mostly

**Comment by seanchelulu** [comment]
I think that it is because the image was already cached. If I am wrong, please correct me. Thanks.

**Reply by kean** [reply, marked as answer]
Hey, yeah. It's going to be `nil` in most scenarios. It's not nil only when the response is coming from the network.
